### ISSUE TITLE
clear log file and write to it with out restarting the server

### ISSUE THREAD

**Issue by zivbrachia** [log4js-node/log4js-node#622]
Hi, I want to manually clear the log file.
I want to use the same features of "maxLogSize" with "backups" and I need the ability of do it manually.
by using log4js command:

1. The log file will be backups it self
2. Change file name 
3. Clear the main log file.

I need your help with solution for this problem, thanks

**Comment by nomiddlename** [comment]
You could call `log4js.shutdown` to stop all logging, then rename the file, then call `log4js.configure` again. That might work.

**Comment by zivbrachia** [comment]
Yes, but I have multiple users with app's. Its not right to stop other
users log because one user

**Comment by nomiddlename** [comment]
Log4js doesn't have any way of doing that. Your app could make sure no logs for that user are being written, copy the log file and then truncate the original (that way log4js can keep the same file handle), then allow logs to be written. But that's beyond the scope of log4js, really.

**Comment by zivbrachia** [comment]
I've managed to manipulate the existing code of log4js of doing so.
Everything is already there.
10x
